examples/multipole.py

```diff
@@ -0,0 +1,145 @@
+# Example illustrating the novel boundary sweeping functions (extrude/revolve) introduced in v0.8.0
+# at the hand of a simple simulation of a multipole
+
+import numpy as np
+
+import traceon.geometry as G
+
+import traceon.solver as S
+import traceon.excitation as E 
+import traceon.plotting as P
+import traceon.tracing as T
+
+
+# multipole configuration
+ORDER = 2 # A second-order multipole is a quadrupole
+THICKNESS = 0.1
+IN_RADIUS = 0.3
+OUT_RADIUS = 0.5
+SPACING = 0.05
+
+# boundary configuration
+B_RADIUS = 1.
+B_HEIGHT = 2.
+
+
+# Before v0.8.0
+# 1. Initialize a path
+# 2. Revolve it
+# 3. Add side pieces
```

Comment:
Please don't show how it was done in the old way, I think that is too much information that would confuse the user, simply use the new way and keep the code nice and short. You can add a comment at the top of the file that functions are being used that are only present in version 0.8